latexify/core.py

```diff
@@ -154,16 +176,16 @@ def visit_If(self, node):
     return latex + self.visit(node) + r', & \mathrm{otherwise} \end{array} \right.'
 
 
-def get_latex(fn):
-  return LatexifyVisitor().visit(ast.parse(inspect.getsource(fn)))
+def get_latex(fn, math_symbol=True):
+  return LatexifyVisitor(math_symbol=math_symbol).visit(ast.parse(inspect.getsource(fn)))
 
 
-def with_latex(fn):
+def with_latex(*args, math_symbol=True):
```

Comment:
This does not work as a decorator. Fix this as:

```
def with_latex(math_symbol=True):
  class _LatexifiedFunction:
    ...
  def _wrapper(fn):
    return _LatexifiedFunction(fn)
  return _wrapper
```

Reply:
Your suggestion will change the behavior so that this happens, which is incompatible with the original API:

![image](https://user-images.githubusercontent.com/53090272/88499887-36853e00-cf95-11ea-8e9a-e4a270fd813f.png)

The original I had supports both `@with_latex(**stuff)` and `@with_latex`:

![image](https://user-images.githubusercontent.com/53090272/88499939-646a8280-cf95-11ea-88cd-8051943bbbe1.png)

The major downside of the approach I used is that the decorator's signature looks ugly, and in some very rare cases (usually when the user is doing something weird and not advisable) it can break.

I believe the authors of [this module](https://github.com/micheles/decorator) came up with a good solution to this problem, but I am trying to not put in any external dependencies.

Reply:
I have implemented the change as per your request, I just want to be clear about what this change will do to the original API.

Reply:
Ah I see, I missed the last case and your impl. is actually better. There's no ploblem to revive the original one.

Reply:
Reimplemented!